include/af/data.h

```diff
@@ -234,11 +234,11 @@ namespace af
 
     /**
         \param[in] in is the input array
-        \param[in] x is the number of times \p in is tiled along first dimension
-        \param[in] y is the number of times \p in is tiled along second dimension
-        \param[in] z is the number of times \p in is tiled along third dimension
-        \param[in] w is the number of times \p in is tiled along fourth dimension
-        \return the tiled output
+        \param[in] x is how many copies of \p in are tiled along the first dimension
+        \param[in] y is how many copies of \p in are tiled along the second dimension
+        \param[in] z is how many copies of \p in are tiled along the third dimension
+        \param[in] w is how many copies of \p in are tiled along the fourth dimension
+        \return The tiled version of the input array
```

Comment:
I think mixture of the old and new statement makes more sense, like
`[ x | y | z ] is the number of times \p in is copied along the [ first | second | third ] dimension` what do you think ?

I think this sits better with your earlier explanation in data.dox file.

Reply:
So I'm aiming to point out that x, y, z, w count actually includes the original. If we say "the number of times it is copied", my first impression would be how many copies (excluding the original) will be produced.

Reply:
I do like that wording though, maybe I'll just include a note on the bottom that points out the inclusivity of the counts